testsuite/forge/src/config.rs

```diff
@@ -238,6 +238,8 @@ impl ForgeConfig {
                 ["enable_storage_sharding"] = true.into();
             helm_values["validator"]["config"]["indexer_db_config"]["enable_event"] = true.into();
             helm_values["fullnode"]["config"]["indexer_db_config"]["enable_event"] = true.into();
+            helm_values["validator"]["config"]["indexer_db_config"]["enable_event_v2_translation"] = true.into();
```

Comment:
This will fail compat because this field doesn't exist in the old version of binary.